tests/impl.cpp

```diff
@@ -1636,7 +1636,20 @@ result_t test_mm_div_ss(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_extract_pi16(const SSE2NEONTestImpl &impl, uint32_t i)
 {
+    // FIXME GCC has bug on `_mm_extract_pi16` intrinsics. We will enable this
```

Comment:
While you were mentioning GCC bug, can you give some reference links? such as GCC bugzilla and/or explicit issues.